Sources/App/Controllers/API/API+PackageCollectionController.swift

```diff
@@ -60,7 +61,7 @@ extension API {
 }
 
 
-extension PackageCollectionModel.V1.Collection: Vapor.Content {}
+extension PackageCollectionSigning.Model.SignedCollection: @retroactive @unchecked Sendable, Vapor.Content {}
```

Comment:
We should open a PR upstream to conform `PackageCollectionModel.V1.Collection` to `Sendable` so we can avoid this `@retroactive @unchecked Sendable` here.